packages/netlify-cms-widget-markdown/src/serializers/index.js

```diff
@@ -7,6 +7,7 @@ import remarkToRehype from 'remark-rehype';
 import rehypeToHtml from 'rehype-stringify';
 import htmlToRehype from 'rehype-parse';
 import rehypeToRemark from 'rehype-remark';
+import registry from 'netlify-cms-core';
```

Comment:
Individually widgets don't have the core as a dependency.

We would need to pass down the plugins as props from here https://github.com/netlify/netlify-cms/blob/3bd36776d6c02a0669df9a5b11f105efee2d2ca3/packages/netlify-cms-core/src/components/Editor/EditorControlPane/Widget.js#L296

Reply:
done.

not totally sure about the [`CopyPasteVisual` plugin](https://github.com/netlify/netlify-cms/blob/adf7ad78a83aba2c258f8c1af2486eb2522e42e0/packages/netlify-cms-widget-markdown/src/MarkdownControl/plugins/CopyPasteVisual.js) and if something needs to be done in [`htmlToSlate`](https://github.com/netlify/netlify-cms/blob/adf7ad78a83aba2c258f8c1af2486eb2522e42e0/packages/netlify-cms-widget-markdown/src/serializers/index.js#L181) as well.

Reply:
I think we don't need it for `htmlToSlate` and the changes to `CopyPasteVisual` look good